Adding a Form in the Contact-Me Section with Validations

hernandanielzamora opened the pull request:
Adding a "form" tag in the contact me section with validations for every input tag. Also added a submit tag to create the button

Commit:
Adding a Form in the Contact-Me Section wicht contains validations for every input tag

fpsapc committed:
add button instead of input field

add button instead of input field and also make button more attactive.

fpsapc commented:
I just add a button in get in touch section instead of input field and also add some transition to buttons so they looks well.

Commit:
Deleted commented sections of the comment - Deleted inline styles applied incorrectly

Commit:
Deleted commented code - Deleted inline styles

Commit:
Deleted Inline Styles - Deleted Commented code

fpsapc committed:
add formspree

add Formspree to my getintouch section

Zuhaib042 commented on styles/styles.css:
```diff
@@ -463,7 +463,6 @@ div ul li {
 
 #getintouch {
   display: grid;
-  width: 100vw;
   height: 100vh;
   background-color: #f5f3f3;
   align-content: center;
```
- [x] Good job on the beautiful design but laying out your element in the form section should be with flex as in the requirements.
  -  In order to lay out the elements on the page you should use Flexbox.

Zuhaib042 commented on index.html:
```diff
@@ -1,234 +1,253 @@
+        <input class="input border" type="email" placeholder="Email address" required>
+        <textarea class="input" name="textarea" id="textarea" cols="30" rows="5" placeholder="Enter text here"
+          maxlength="500" required></textarea>
+          <div class="centerbtn"><button type="submit" class="getintouchbtn work-button">Get in touch</button>
```
- [x] Kindly consider applying the `pressed` state in button interactions as suggested in the figma design by using the `:active` pseudoclass as our design need to be similar to the figma design.
figma design
![er](https://user-images.githubusercontent.com/96683843/220424376-8de8a685-2a2c-4fe4-9f23-928311129856.jpg)

Zuhaib042 commented on index.html:
```diff
@@ -1,234 +1,253 @@
+  </main>
+
+  <footer>
+    <div class="sociallinks">
+      <ul class="footercenter">
+        <li>
+          <a href="https://github.com/fpsapc" target="_blank" rel="noopener"> <i
+              class="fa-brands fa-github social"></i></a>
+        </li>
+        <li>
+          <a href="https://linkedin.com/in/salman-ahmad1987" target="_blank" rel="noopener"> <i
+              class="fa-brands fa-linkedin social"></i></a>
+        </li>
+        <li>
+          <a href="https://www.linkedin.com/company/vivimedia/about/" target="_blank" rel="noopener"> <i
+              class="fa-brands fa-square-facebook social"></i></a>
+        </li>
+        <li>
+          <a href="https://twitter.com/salman-ahmadkhan1987" target="_blank" rel="noopener"> <i
+              class="fa-brands fa-twitter social"></i></a>
+        </li>
+        <li>
+          <a href="mailto:fpsapc@gmail.com" target="_blank" rel="noopener"> <i
+              class="fa-sharp fa-solid fa-m social"></i></a>
+        </li>
+      </ul>
+    </div>
```
- [x] Kindly consider using the social icons from the figma design as sticking to the figma design as much as possible is one of the project requirements.kindly consider downloading the figma design social icons to use.
figma design
![fd](https://user-images.githubusercontent.com/96683843/220425008-fef1f15e-a185-4a67-aed4-af871b4fb591.jpg)

your design
![as](https://user-images.githubusercontent.com/96683843/220425029-a4a52edf-25d3-46fd-941c-b44e0e87206d.jpg)

Zuhaib042 reviewed (changes requested):
Hi @hernandanielzamora  @fpsapc , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![close](https://media.giphy.com/media/3o7qDQAFlF8mZjv6fK/giphy.gif)

## Highlights
✅ Correct Github flow


## Required Changes ♻️
- [x] kindly consider adding @hernandanielzamora as the second author in your readme as it will make your readme more professional.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

fpsapc committed:
add Hernan as author

Add Hernan as a author as code reviewer suggests.

fpsapc committed:
use flexbox instead of grid

use flexbox instead of grid in my getintouch section as suggested by the reviewer

fpsapc committed:
add social icon from figma templete

add social icons from Figma template and style them accordingly.

fpsapc commented:
I also follow the buttons design in the last commit.

souad988 reviewed (changes requested):
Hi @hernandanielzamora , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![212385281-4797c58a-d358-423a-aa7c-832c22203865](https://user-images.githubusercontent.com/59707859/220454973-375dad95-a929-4210-a602-a61436c628be.gif)


## Required Changes ♻️
_Very well done on implementing all requested changes by previous reviewer Only one thing before you can move to next project , your footer is overlapping the form you need to fix this issue 

**your design:**
![student contact form temp2](https://user-images.githubusercontent.com/59707859/220455553-c6dba0f6-9a6f-45bf-832b-9757d390a321.PNG)

Please feel free to ask if you need help 😊

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

fpsapc committed:
fix overlapping issue

I remove the height property from sections to fix the overlapping issue.

fpsapc committed:
remove width property

remove width property applied to main container

binhussen reviewed (approved):
Hi @fpsapc , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉
<img src="https://user-images.githubusercontent.com/65852621/133696182-6c5b784b-0083-427a-b24d-3ec870915f79.jpg"/>
## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
